### ISSUE TITLE
More squelch levels ????

### ISSUE THREAD

**Issue by sp2ong** [rogerclarkmelbourne/radioddity_gd-77_cps#35]
Is it possible add in Community release more levels for squelch ???
At current we have only 2 levels "Normal" and "Strict".
For Analog mode it will be useful has more levels because in many cases NORMAL level squelch is still too high.

**Comment by SQ9MDD** [comment]
I think it's hardware depended...

**Comment by DL2MF** [comment]
Like in other projects, this should be kept as an issue tracker, no support or feature request forum.

**Comment by rogerclarkmelbourne** [comment]
Someone posted the same question as comment on my blog.

AFIK there is no way to add more squelch levels as the dropdown menu is stored in a single binary bit in the codeplug data that is sent to the GD77

I discussed this with @g4eml , and he said there was another byte of data somewhere in the codeplug which he thought had something to do with squelch, but AFIK it had no effect when he changed it.

**Comment by g4eml** [comment]
Hi Roger, 

The 9 level squelch setting is coded in the CPS and is read from and written to the codeplug, but the controls on the channel form are set to invisible as are the ones for Tx reference frequency and RX reference frequency.  

I did try enabling the RX reference frequency one as I have one channel that suffers from an internally generated spurious signal and I wanted to see if I could move it. However although I could set the value and write it to the radio it didn’t make any difference. So I can only assume they are set to invisible because they do not work on this radio. Maybe they do on another model or firmware. 

73 

Colin. 



On 25 Feb 2018, at 20:49, Roger Clark <notifications@github.com> wrote:

Someone posted the same question as comment on my blog.

AFIK there is no way to add more squelch levels as the dropdown menu is stored in a single binary bit in the codeplug data that is sent to the GD77

I discussed this with @g4eml , and he said there was another byte of data somewhere in the codeplug which he thought had something to do with squelch, but AFIK it had no effect when he changed it.
